tests/test_utils_misc.py

```diff
@@ -18,6 +19,17 @@ def test_non_ascii_unicode(self):
     def test_non_ascii_bytes(self):
         assert get_crc32(u'example\u5000'.encode('utf8')) == 1259721235
 
+    def test_negative_crc32(self):
+        assert get_crc32(b'1') == -2082672713
+
+    def test_crc32_range(self):
```

Comment:
Np! :)
Yep, both these tests fail without ``to_signed32`` under py35 for me (I fixed the py35 test in 4a10c08, didn't notice it was failing at first).
```
(py35) frontera (git::sql-crc3-int-type) $ git diff
diff --git a/frontera/utils/misc.py b/frontera/utils/misc.py
index f7372bc..b1c7933 100644
--- a/frontera/utils/misc.py
+++ b/frontera/utils/misc.py
@@ -44,6 +44,7 @@ def get_crc32(name):
 def to_signed32(x):
     """ If x is an usigned 32-bit int, convert it to a signed 32-bit.
     """
+    return x
     return x - 0x100000000 if x > 0x7fffffff else x
 
 
(py35) frontera (git::sql-crc3-int-type) $ py.test tests/test_utils_misc.py::TestGetCRC32
============================================================================== test session starts ===============================================================================
platform darwin -- Python 3.5.1, pytest-2.9.2, py-1.4.31, pluggy-0.3.1
rootdir: /Users/kostia/shub/frontera, inifile: tox.ini
collected 6 items 

tests/test_utils_misc.py ....FF
============================================================================ short test summary info =============================================================================
FAIL tests/test_utils_misc.py::TestGetCRC32::()::test_negative_crc32
FAIL tests/test_utils_misc.py::TestGetCRC32::()::test_crc32_range

==================================================================================== FAILURES ====================================================================================
________________________________________________________________________ TestGetCRC32.test_negative_crc32 ________________________________________________________________________

self = <tests.test_utils_misc.TestGetCRC32 object at 0x105e20d30>

    def test_negative_crc32(self):
>       assert get_crc32(b'1') == -2082672713
E       assert 2212294583 == -2082672713
E        +  where 2212294583 = get_crc32(b'1')

tests/test_utils_misc.py:23: AssertionError
_________________________________________________________________________ TestGetCRC32.test_crc32_range __________________________________________________________________________

self = <tests.test_utils_misc.TestGetCRC32 object at 0x105e17e80>

    def test_crc32_range(self):
        left, right = -2**31, 2**31 - 1
        for x in range(10000):
            bytestr = hashlib.md5(str(x).encode('ascii')).hexdigest()
>           assert left <= get_crc32(bytestr) <= right
E           assert 2157876746 <= 2147483647
E            +  where 2157876746 = get_crc32('a87ff679a2f3e71d9181a67b7542122c')

tests/test_utils_misc.py:29: AssertionError
======================================================================= 2 failed, 4 passed in 0.12 seconds =======================================================================
```